tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -102,16 +103,24 @@ public function userSearchesUsingWebDavAPI(
 			$body .= "	</a:prop>";
 		}
 		$body .= "	</oc:search-files>";
-		$response = WebDavHelper::makeDavRequest(
-			$baseUrl,
+		$davPathVersionToUse = $this->featureContext->getDavPathVersion();
+		$davPath = WebDavHelper::getDavPath($doDavRequestAsUser ?? $user, $davPathVersionToUse, 'files');
+
+		if ($davPathVersionToUse == WebDavHelper::DAV_VERSION_NEW) {
+			// Removes the last component('username' in this case) from the WebDAV path by going up one level in the directory structure.
+			// e.g. remote.php/dav/files/Alice ==> remote.php/dav/files/
+			$davPath = \dirname($davPath, 1) . '/';
+		}
+
+		$fullUrl = WebDavHelper::sanitizeUrl($baseUrl . "/$davPath" . '/');
```

Comment:
maybe we don't need the trailing slash
```suggestion
			$davPath = \dirname($davPath, 1);
		}

		$fullUrl = WebDavHelper::sanitizeUrl("$baseUrl/$davPath");
```